examples/mailersend/package.json

```
@@ -4,23 +4,24 @@
   "license": "MIT",
   "private": true,
   "sideEffects": false,
+  "type": "module",
   "main": "./dist/index.js",
-  "module": "./dist/index.mjs",
-  "types": "./dist/index.d.ts",
   "files": [
     "dist/**"
   ],
   "scripts": {
-    "build": "tsup src/index.tsx --format esm,cjs --dts --external react",
-    "dev": "tsup src/index.tsx --format esm,cjs --dts --external react --watch",
+    "build": "tsup-node src/index.tsx --format esm --target node18",
+    "dev": "tsup-node src/index.tsx --format esm --target node18 --watch",
     "clean": "rm -rf dist"
   },
   "dependencies": {
-    "@react-email/components": "*",
+    "@react-email/components": "workspace:0.0.25",
```

```suggestion
    "@react-email/components": "workspace:*",
```

do we need the specific version?

---

I think it'd be better with a specific version, thinking it's better when trying to compare the example to their project.  We also don't need to worry about the dependency updating, since changeset should do it for us